frontend/src/pages/videos/VideoRecommendation.tsx

```diff
@@ -43,27 +43,30 @@ function VideoRecommendationPage() {
   }, [location.search]);
 
   return (
-    <ContentBox noMinPadding maxWidth="lg">
-      <Box px={{ xs: 2, sm: 0 }}>
-        <SearchFilter />
-      </Box>
-      <LoaderWrapper isLoading={isLoading}>
-        <VideoList
-          videos={videos.results || []}
-          emptyMessage={
-            isLoading ? '' : t('noVideoCorrespondsToSearchCriterias')
-          }
-        />
-      </LoaderWrapper>
-      {!isLoading && videoCount > 0 && (
-        <Pagination
-          offset={offset}
-          count={videoCount}
-          onOffsetChange={handleOffsetChange}
-          limit={limit}
-        />
-      )}
-    </ContentBox>
+    <>
+      <ContentHeader title={t('recommendationsPage.contentHeader')} />
```

Comment:
I just added `<>` and `<ContentHeader>` here. The indentation make a good part of the file to appear as new to Git.